Exercises.IEnumerable2/Dependences/ItemInWidthCollection.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+namespace Exercises.IEnumerable2.Dependences
+{
+    public class ItemInWidthCollection : IEnumerable<Item>, IEnumerator<Item>
```

Comment:
`IEnumerable` is a factory which should create `IEnumerator` on each call of `GetEnumerator()`. Executing several enumerators with different enumeration position should be supported. Not critical requirement, but when writing generic enumerator all use cases should be covered